deel-2-flexbox-layout/opdracht-5/styles/styles.css

```diff
@@ -1 +1,59 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body {
+    display: flex;
+    width: 100vw;
+    height: 100vh;
+    flex-direction: column;
+}
+
+header{
+    display: flex;
+    flex-direction: row;
+    flex-basis: 150px;
+    order: 0;
```

Comment:
als je default order goed is, hoef je deze property niet expliciet toe te voegen.